polyfill/test/Absolute/prototype/minus/order-of-operations.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.absolute.prototype.minus
+includes: [compareArray.js]
+---*/
+
+const instance = new Temporal.Absolute(10n);
+const expected = [
+  "get days",
+  "valueOf days",
+  "get hours",
+  "valueOf hours",
+  "get microseconds",
+  "valueOf microseconds",
+  "get milliseconds",
+  "valueOf milliseconds",
+  "get minutes",
+  "valueOf minutes",
+  "get months",
+  "get nanoseconds",
+  "valueOf nanoseconds",
+  "get seconds",
+  "valueOf seconds",
+  "get years",
+];
+const actual = [];
+const fields = {
+  days: 1.7,
+  hours: 1.7,
+  minutes: 1.7,
+  seconds: 1.7,
+  milliseconds: 1.7,
+  microseconds: 1.7,
+  nanoseconds: 1.7,
+};
+const argument = new Proxy(fields, {
+  get(target, key) {
+    actual.push(`get ${String(key)}`);
```

Comment:
nitpick: Throughout the tests, I think `String()` is redundant here.

Reply:
I thought I'd hit a `Symbol` here during development, but I cannot reproduce it now. Changed.